tests/robot/ADMIN_TESTS/006-Cache.robot

```diff
@@ -0,0 +1,394 @@
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+*** Settings ***
+Metadata    Version    0.1.0
+Metadata    Authors    *Wladislaw Wagner, Stefan Spiska*
+Metadata    Created    2021.07.13
+Metadata    Updated    2021.08.09
+Metadata    Command    robot -d results -L TRACE -i cache robot/ADMIN_TESTS
+Metadata    TOP_TEST_SUITE    CACHE
+
+Documentation    Tests related to caching \n\n
+...              How to run these tests locally \n\n
+...              ============================== \n\n
+...
+...              robot -d results/ -L TRACE -i CACHE robot/ADMIN_TESTS/
+...              OR
+...              robot -v SUT:ADMIN-TEST -d results/ -L TRACE -i cache robot/ADMIN_TESTS/
+...              OR (when you have started EHRbase + DB manually)
+...              robot -v SUT:ADMIN-DEV -d results/ -L TRACE -i cache robot/ADMIN_TESTS/
+...
+...              Thanks to Stefan Spiska for providing the documentation for this test in \n\n
+...              https://github.com/ehrbase/project_management/issues/451
+
+Resource        ../_resources/keywords/admin_keywords.robot
+Resource        ../_resources/keywords/ehr_keywords.robot
+Resource        ../_resources/keywords/composition_keywords.robot
+Resource        ../_resources/keywords/template_opt1.4_keywords.robot
+Resource        ../_resources/keywords/aql_query_keywords.robot
+
+Suite Setup     startup SUT
+Suite Teardown  shutdown SUT
+
+Force Tags     cache    cache_template_update
+
+
+
+*** Variables ***
+# comment: overriding defaults in suite_settings.robot
+${SUT}                         ADMIN-TEST
+${CACHE-ENABLED}               ${TRUE}
+${ALLOW-TEMPLATE-OVERWRITE}    ${FALSE}
+
+${VALID DATA SETS}     ${PROJECT_ROOT}${/}tests${/}robot${/}_resources${/}test_data_sets${/}valid_templates
+
+
+
+*** Test Cases ***
+Commit Composition After Template Update (Cache Enabled)
+    [Tags]    
+    
+    01) Run ehrbase with cache enabled true
+    02) Upload Template Containing Node With Specific Archetype_ID
+    03) Upload Composition Containing Entity Corresponding To Specific Archetype_ID
+    04) Conduct Checks via 'GET Opt' and 'AQL Queries'
+    05) Update Template Changing The Specific Archetype_ID To NEW Archetype_ID
+    06) Upload Composition (To Updated Template) Containing Entity Corresponding To NEW Specific Archetype_ID
+    07) Conduct Checks via 'GET Opt' and 'AQL Queries'
+    08) Restart ehrbase
+    09) Conduct Checks To Validate Cache Works Correctly
+    # 10. Check That: Get Opt / Get Web-Template Contains The NEW Specific Archetype_ID    # TODO: clarify w/ @Stefan
```

Comment:
@stefanspiska in your [issue](https://github.com/ehrbase/project_management/issues/451) step 10 seems to be a duplicate of step 9